inspect.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+
+	"github.com/containerd/containerd/namespaces"
+	"github.com/genuinetools/img/client"
+	"github.com/moby/buildkit/identity"
+	"github.com/moby/buildkit/session"
+)
+
+const inspectUsageShortHelp = `Return low-level information about the image.`
+const inspectUsageLongHelp = `Return low-level information about the image.`
```

Comment:
Can we clarify in the help string that the output format is incompatible with `docker image inspect`?

Reply:
Absolutely. I'll make that change.

Specifically the output is the OCI Image configuration standard and not the Docker standard. It would be possible if we wanted in another change to detect the Docker one and show that instead but I wanted to limit the complexity and keep it standards-based if possible.

Reply:
Done